### ISSUE TITLE
Issue with libs?

### ISSUE THREAD

**Issue by pdefreitas** [openpeer/ortc-lib#43]
I see that the project as switched to gclient. But I'm facing issues sync'ing the project.

`C:\workspace\ortc-lib\libs>gclient sync
Oh hai! You are using subversion. Chrome infra is eager to get rid of
svn support so please switch to git.
Tracking bug: http://crbug.com/475320
If you are a project owner, you may request git migration assistance at:
  https://code.google.com/p/chromium/issues/entry?template=Infra-Git

________ running 'svn checkout file:///./webrtc C:\workspace\ortc-lib\libs\webrtc --non-interactive --ignore-externals' in 'C:\workspace\ortc-lib\libs'
svn: Unable to open an ra_local session to URL
svn: Unable to open repository 'file:///webrtc'

webrtc (ERROR)
----------------------------------------
[0:00:00] Started.
----------------------------------------
Error: Command 'svn checkout file:///./webrtc C:\\workspace\\ortc-lib\\libs\\webrtc --non-interactive --ignore-externals' returned non-zero exit status 1 in C:\workspace\ortc-lib\libs`

**Comment by robin-raymond** [comment]
@pdefreitas No, we aren't using gclient. This is designed to be a recursive clone + script run and open and build, nothing more needed. We are actively working to improve and fix any script issues but we do not maintain a full webrtc within ortc as it's a subset only using the API sets we require.
